account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -783,6 +783,22 @@ def fields_to_ignore(self, name):
         } | specials_mapping.get(name, set())
         return set(models.MAGIC_COLUMNS) | specials
 
+    @api.model
+    @tools.ormcache("name")
```

Comment:
This is not an api.model function because it does need a record.

Besides, caching it won't save much time because the things it does are quite cheap.